.github/workflows/ci.yml

```diff
@@ -456,7 +450,7 @@ jobs:
   build_test_runtime_windows:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'build_test_runtime_windows')
-    runs-on: windows-2022-64core
+    runs-on: windows-2022
```

Comment:
3 minutes -> 17 minutes for this job. Not great, but we can live with it.

* before: https://github.com/iree-org/iree/actions/runs/8806894791/job/24172720432
* after: https://github.com/iree-org/iree/actions/runs/8806704450/job/24172131019?pr=17145
  * Seems like ~8 minutes is spent linking test executables. Reworking how the HAL CTS tests are defined would help there.

Reply:
Some ideas for cutting down on that linking time are here: 
* https://github.com/iree-org/iree/pull/16273#issuecomment-1918174524
* https://discord.com/channels/689900678990135345/689906000043573354/1232438158105509938